### PR TITLE
[VEN-2882] Deploy Multichain Governance on Base Sepolia

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves #

### CODE REVIEW

File: deploy/008-configure-acm-commands-aggregator.ts [hunk 1]
```diff
@@ -348,6 +379,20 @@ const revokePermissions: Permissions = {
...
+    ...getPrimeLiquidityProviderRevokePermissions(BASESEPOLIA_PLP, BASESEPOLIA_GUARDIAN),
+    ...getResilientOracleRevokePermissions(BASESEPOLIA_RESILIENT_ORACLE, BASESEPOLIA_GUARDIAN),
+    ...getBoundValidatorRevokePermissions(BASESEPOLIA_BOUND_VALIDATOR, BASESEPOLIA_GUARDIAN),
+    ...getXVSVaultRevokePermissions(BASESEPOLIA_XVS_VAULT_PROXY, BASESEPOLIA_GUARDIAN),
+    ...getRewardDistributorRevokePermissions(BASESEPOLIA_GUARDIAN, true),
+    ...getIRMRevokePermissions(BASESEPOLIA_GUARDIAN),
+    ...getPoolRegistryRevokePermissionsForWildcard(BASESEPOLIA_GUARDIAN),
+    ...getComptrollerRevokePermissions(BASESEPOLIA_GUARDIAN),
+    ...getVTokenRevokePermissions(BASESEPOLIA_GUARDIAN),
+    ...getOmniChainExecutorOwnerRevokePermissions(BASESEPOLIA_OMNICHAIN_EXECUTOR_OWNER, BASESEPOLIA_GUARDIAN),
```
Comment (GitGuru7) [inline deploy/008-configure-acm-commands-aggregator.ts:393]: I believe we are expected to revoke these permissions from the guardian once multichain governance is fully assured.

File: deploy/008-configure-acm-commands-aggregator.ts [hunk 2]
```diff
@@ -257,6 +270,24 @@ const grantPermissions: Permissions = {
...
+    ...getXVSVaultPermissions(BASESEPOLIA_XVS_VAULT_PROXY),
+    ...getPoolRegistryPermissions(BASESEPOLIA_POOL_REGISTRY),
+    ...getPrimePermissions(BASESEPOLIA_PRIME),
+    ...getPrimeLiquidityProviderPermissions(BASESEPOLIA_PLP),
+    ...getProtocolShareReservePermissions(BASESEPOLIA_PSR),
+    ...getComptrollerPermissions(),
+    ...getVTokenPermissions(),
+    ...getRewardDistributorPermissionsTimebased(),
+    ...getIRMPermissions(),
+    ...getOmniChainExecutorOwnerPermissions(BASESEPOLIA_OMNICHAIN_EXECUTOR_OWNER, BASESEPOLIA_GUARDIAN),
```
Comment (GitGuru7) [inline deploy/008-configure-acm-commands-aggregator.ts:289]: I believe we don't need this here. We've addressed it in the past three testnets to ensure compatibility and alignment with mainnet permissions, but we don't need them anymore. 

I think we should move `getOmniChainExecutorOwnerPermissions` to this file only as we will not need them in future.

File: deploy/008-configure-acm-commands-aggregator.ts (hunk 1 above)
Comment (chechu) [reply deploy/008-configure-acm-commands-aggregator.ts:392]: Those permissions were also revoked on `opbnbtestnet`, `sepolia`, `arbitrumsepolia`, with the rest of the revokes. But they were not revoked for `zksyncsepolia` or `opsepolia`. Those permissions were not revoked in any mainnet yet.

So, I would not revoke those permissions on `basesepolia` yet (I would remove that line from the configuration). And I would revoke those permissions in the future.

```suggestion
```

Comment (GitGuru7) [reply deploy/008-configure-acm-commands-aggregator.ts]: We have revoked those permissions on `opbnbtestnet,` `sepolia`,` arbitrumsepolia`  just to ensure compatibility and alignment with mainnet permissions. But we don't need them anymore as, we already have similar permissions set for mainnet and testnet on the networks like `zksync` and `opBNB` testnets.

**Review by chechu — APPROVED** [review]

**Pull request merged**